packages/ui-components/src/Shared.styles.ts

```diff
@@ -21,17 +21,12 @@ import {
   WrapperDivProps,
 } from './StyleProps';
 
-export const MeasureApp = styled.div`
-  display: flex;
-  width: 100%;
-  padding: 0 calc(50vw - 600px);
-
-  @media (max-width: 1280px) {
-    padding: 0 40px;
-  }
+export const ContainerFlex = styled(SUIContainer)`
+  display: flex !important;
```

Comment:
A question: Can you mix styled-components with tachyons? Something like this would be cool:

```ts
export const ContainerFlex = styled(SUIContainer)`
  flex items-center // some other tachyon classes
`
```

Reply:
Answering myself after some googling: https://github.com/jxnblk/tachyons-components/issues/1#issuecomment-333703650.

so maybe something like

```ts
export const ContainerFlex = styled(SUIContainer).attr({
  'flex items-center'
})`;
```

If that works, I would be advocate to use tachyon classes where possible in ui-components

Reply:
This might won't always work with rewriting the semantic UI css, however 👍will bulk that in the next PR